src/lib/routes/admin-api/environments.ts

```diff
@@ -120,8 +137,12 @@ export class EnvironmentsController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Environments'],
+                    description: 'Toggles the environment with `name` on',
```

Comment:
Is this enable/disable environments for the global unleash instance or for projects? If the former, is it whether an env should be deprecated or not? 

I think the description could be a bit clearer on what this does. The operation is also missing a summary.